### DISCUSSION TITLE
PR changes don't reflect immediately in MDN Docs

### DISCUSSION THREAD

**Discussion by PositiveUser** [mdn/mdn-community#500] [Content] [answered]
Why don't I see changes being reflected in MDN pages after PRs get merged?

Do the maintainers manually update the live site after the pull requests get merged?

**Comment by Josh-Cena** [comment, marked as answer]
The site is automatically deployed every 24 hours, so it will take no longer than 48 hours for your change to go live.

**Comment by PositiveUser** [comment]
But how does it update automatically every 24 hours? What happens behind the scenes? How does the site source data from a repository? What is the site hosted on?

**Reply by Josh-Cena** [reply]
This would be better answered by @mdn/core-dev, but long story short, the site is hosted on a Google cloud service, and there's a cron job that pulls in the repositories' sources every day to build and deploy.
